### ISSUE TITLE
Not showing many to many relationship.

### ISSUE THREAD

**Issue by TheDailySpank** [beyondcode/laravel-er-diagram-generator#5]
I have two models connected via pivot table in a many to many relationship where any number of webpages can be connected to any number of images. Setup as per the docs here: https://laravel.com/docs/5.6/eloquent-relationships#many-to-many

All other relations display on the graph as expected, which is very nice, thank you!

App\Image.php
```
    /**
     * @return mixed
     */
    public function webpages()
    {
        return $this->belongsToMany(Webpage::class)->withTimestamps();
    }
```

App\Webpage.php
```
    /**
     * @return mixed
     */
    public function images()
    {
        return $this->belongsToMany(Image::class)->withTimestamps();
    }
```

Any idea what I am missing?

**Comment by mpociot** [comment]
I just published a new release (0.2.1) which fixes this issue.

**Comment by TheDailySpank** [comment]
Awesome! It works now.

Closing.
